tableauserverclient/server/server.py

```diff
@@ -47,6 +58,29 @@ def _set_auth(self, site_id, user_id, auth_token):
         self._user_id = user_id
         self._auth_token = auth_token
 
+    def _get_legacy_version(self):
+        response = self._session.get(self.server_address + "/auth?format=xml")
+        info_xml = ET.fromstring(response.content)
+        prod_version = info_xml.find('.//product_version').text
+        version = _PRODUCT_TO_REST_VERSION.get(prod_version, '2.1')  # 2.1
```

Comment:
I'll make that comment not a random repetition: `2.1 is the version with permissions changes`